packages/slate-editor/src/lib/hooks/useFunction.ts

```diff
@@ -6,7 +6,7 @@ import { useLatest } from './react-use';
  * Return a stable proxy-function
  * that will always call the latest version of the input function.
  */
-export function useFunction<F extends () => any>(func: F): F {
+export function useFunction<F extends (...params: any) => any>(func: F): F {
```

Comment:
Could you use `...params: never[]`
because if you will forgive to set type it will be `any` without errors
<img width="746" alt="image" src="https://user-images.githubusercontent.com/3620639/178267204-516c9f01-e3b7-468a-b266-c997a18f483f.png">